recipes/piler/meta.yaml

```diff
@@ -0,0 +1,33 @@
+package:
+  name: piler
+  version: "0.1"
+
+build:
+  number: 0
+
+source:
+  fn:  piler_source.tar.gz # [linux]
+  url: http://www.drive5.com/piler/piler_source.tar.gz # [linux]
+  md5: 6f351fb437d0cc6a1ec84f41998621c9 # [linux]
+  fn:  piler_pals_osx_src.tar # [osx]
+  url: http://drive5.com/piler/piler_pals_osx_src.tar.gz # [osx]
+  md5: 8a70e346a45d9f0531642e0ee17e5d86 # [osx]
+
+requirements:
+  build:
+  - gcc # [linux]
+  - llvm # [osx]
+
+  run:
+  - libgcc # [linux]
+
+test:
+  commands:
+    - piler2 -version # [linux]
```

Comment:
Uhh, why is that?
I don't think it is good to have different behaviour for different OS. Can we symlink the one to the other and make it consistent?